.github/workflows/test.yml

```diff
@@ -9,13 +9,13 @@ on:
     branches: ['*']
 
 env:
-  XCODE_VERSION: "13.0"
+  XCODE_VERSION: "14.0"
```

Comment:
Did version 13, stop from building locally? or is it just good to be on the latest version?